### PR TITLE
DeploymentList item got disappear when HomePage re-render

### PR DESCRIPTION
**Pull request by trangcongthanh**
In this PR, I fixed:
- DeploymentList item got disappear when HomePage re-render.
- Typo on Table header Sate -> State.

Also added colors for each deployment state.

### CODE REVIEW

File: admin/src/view/index.js [hunk 1]
```diff
@@ -122,7 +127,9 @@ const DeploymentLast = (props) => {
           </div>
           <div className="mb-4">
             <div className="label">STATE</div>
-            <div>{deployment.readyState === 'READY' ? `READY (${moment(deployment.ready).format(DATETIME_FORMAT)})` : deployment.readyState}</div>
+            <StatusWrapper state={deployment.readyState}>
```
Comment (f3l1x) [inline admin/src/view/index.js:130]: Date is missing here.

**Comment by f3l1x** [comment]
Hi @trangcongthanh. Thank you for PR. Could you please send here a screenshot? I would like to see the difference. :-)

File: admin/src/view/index.js (hunk 1 above)
Comment (trangcongthanh) [reply admin/src/view/index.js:130]: I saw it same as CREATED at below this field. Why you need to duplicate that info?

Or maybe its your mistake xD I can fix it in this PR if its true.

**Comment by trangcongthanh** [comment]
> Hi @trangcongthanh. Thank you for PR. Could you please send here a screenshot? I would like to see the difference. :-)

Sure :D 
Here the screenshot.
image

File: admin/src/view/index.js (hunk 1 above)
Comment (f3l1x) [reply admin/src/view/index.js:130]: It was intentional. There is created dated and ready state date. You can see how long it takes to deploy. What do you think?

Comment (trangcongthanh) [reply admin/src/view/index.js:130]: I knew the purpose. But current implement Created and State was formatted from same value of `deployment.ready`.

Created should be formatted from `deployment.createdAt`.

Or if you want to show how long it takes:
`moment.duration(deployment.ready - deployment.createdAt)`

Comment (f3l1x) [reply admin/src/view/index.js:130]: Oh, you're right. So, please fix CREATED to use `deployment.createdAt` and in STATE keep `deployment.ready`. Thx.

Comment (trangcongthanh) [reply admin/src/view/index.js]: The screenshot after changed

image

**Comment by f3l1x** [comment]
Thank you

**Pull request merged**